firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -388,9 +393,27 @@ uint8_t TCMDEXEC_heartbeat_on(const char *args_str, TCMD_TelecommandChannel_enum
 
 uint8_t TCMDEXEC_core_system_stats(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
                         char *response_output_buf, uint16_t response_output_buf_len) {
-    // TODO: implement this (Issue #103)
-    // Use `TCMD_get_agenda_used_slots_count`
-    snprintf(response_output_buf, response_output_buf_len, "System stats: TODO\n");
+
+    // TODO: Add temperatures (EPS, OBC, antenna, etc.)
+    // TODO: Add beacon sent count
+
+    char timestamp_string[20];
+    GEN_uint64_to_str(TIM_get_current_unix_epoch_time_ms(),timestamp_string);
+
+    char time_of_last_tcmd_sent_string[20];
+    GEN_uint64_to_str(TCMD_latest_received_tcmd_timestamp_sent, time_of_last_tcmd_sent_string);
+    
+    snprintf(response_output_buf, response_output_buf_len, "{\"timestamp\":\"%s\",\"uptime\":\"%lu\",\"last_resync_ms\":\"%lu\",\"delta_uptime\":\"%lu\",\"time_of_last_tcmd_sent\":\"%s\",\"total_tcmd_count\":\"%lu\",\"fs_mount_status\":\"%u\",\"last_time_sync_source\":\"%c\",\"reboot_reason\":\"%s\"}\n" ,
+    timestamp_string, // timestamp
+    TIM_get_current_system_uptime_ms(), // uptime
+    TIM_system_uptime_at_last_time_resync_ms, // last_resync_ms
+    TIM_get_current_system_uptime_ms() - TIM_system_uptime_at_last_time_resync_ms, // delta_uptime
```

Comment:
Suggestion: rename `delta_uptime` to `time_synced_ms_ago`, or something like that.